tools/astarte_e2e/lib/astarte_e2e/config.ex

```diff
@@ -177,6 +185,16 @@ defmodule AstarteE2E.Config do
     os_env: "E2E_MAIL_SERVICE",
     type: BambooMailAdapter
 
+  @envdoc "The JWT public key."
+  app_env :jwt_public_key_pem, :astarte_e2e, :jwt_public_key_pem,
+    os_env: "E2E_REALM_PUBLIC_KEY_PEM",
```

Comment:
the end to end tests won't run if the jwt is not specified

```suggestion
    os_env: "E2E_REALM_PUBLIC_KEY_PEM",
    required: true,
```